lib/java/test-utils/src/main/java/org/enso/test/utils/ContextUtils.java

```diff
@@ -35,7 +35,7 @@ public static Context createDefaultContext() {
     return context;
   }
 
-  public static Context createDefaultContext(OutputStream out) {
```

Comment:
Could we replace the static methods in `ContextUtils` by:
```
var ctx = ContextRule.create()
ctx.eval(...)
ctx.close();
```
e.g. use `ContextRule` as an `AutoClosable` resource?

Reply:
On 952b6752b2254a65fe63107c919021824066cb6c, `ContextUtils` implements `Autoclosable`.